distribution/src/main/resources/licenses/antlr-bsd.txt

```diff
@@ -0,0 +1,11 @@
+Copyright (c) 2012 Terence Parr and Sam Harwell
```

Comment:
Just a thought - perhaps we could download this file during the build? Otherwise we'd need to maintain all header information (copyright year etc).